src/extensions/events.py

```diff
@@ -15,10 +16,13 @@ def __init__(self, bot):
     if message.author != self.bot.user:
       if 'Moderator' not in message.author.roles:
         pattern = 'discord.gg/[a-z0-9A-Z]+'
-         if re.search(pattern, message.content)
+         if re.search(pattern, message.content):
```

Comment:
Add comments to explain the process of what the bot is doing. First he deletes, then messages, then kicks etc.